src/widgets/forms/basic.ts

```diff
@@ -0,0 +1,235 @@
+import { st, Literal, Node, NamedNode } from 'rdflib'
+import store from '../../store'
+import ns from '../../ns'
+import { textInputStyle } from '../../style'
+import { label } from '../../utils'
+
+import { errorMessageBlock } from '../error'
+import { mostSpecificClassURI } from './fieldFunction'
+import { fieldParams } from './fieldParams'
+
+/**
+ * Create an anchor element with a label as the anchor text.
+ *
+ * @param dom The DOM
+ * @param property href for the anchor element
+ * @param fieldInQuestion field to produce a label for
+ *
+ * @ignore exporting this only for unit tests
```

Comment:
Any reason why you don't use @internal?

Reply:
ah right I keep forgetting which one is which :)